eZ/Bundle/EzPublishCoreBundle/Resources/config/routing/internal.yml

```diff
@@ -1,13 +1,3 @@
-# Internal routes
-_ezpublishLocation:
```

Comment:
Please mention in the BC doc (`doc/bc/changes-8.0.md`) in the `Removed features` section that this route has been removed.